src/lib/ccf/index.ts

```diff
@@ -0,0 +1,458 @@
+        consumption: {
+          idle: parseFloat(instance['Instance @ Idle'].replace(',', '.')),
+          tenPercent: parseFloat(instance['Instance @ 10%'].replace(',', '.')),
+          fiftyPercent: parseFloat(
+            instance['Instance @ 50%'].replace(',', '.')
+          ),
+          hundredPercent: parseFloat(
+            instance['Instance @ 100%'].replace(',', '.')
+          ),
+          minWatts: minWatts,
+          maxWatts: maxWatts,
+        },
+        vCPUs: cpus,
+        maxvCPUs: parseInt(instance['Platform Total Number of vCPU'], 10),
+        name: instance['Instance type'],
+      } as IComputeInstance;
+    });
+    gcp_instances.forEach((instance: KeyValuePair) => {
+      const cpus = parseInt(instance['Instance vCPUs'], 10);
+      let architecture = instance['Microarchitecture'];
+      if (!(architecture in this.azureList)) {
+        architecture = 'Average';
+      }
+      this.computeInstances['gcp'][instance['Machine type']] = {
+        name: instance['Machine type'],
+        vCPUs: cpus,
+        consumption: {
+          minWatts: this.gcpList[architecture]['Min Watts'] * cpus,
+          maxWatts: this.gcpList[architecture]['Max Watts'] * cpus,
+        },
+        maxvCPUs: parseInt(
+          instance['Platform vCPUs (highest vCPU possible)'],
+          10
+        ),
+      } as IComputeInstance;
+    });
+    azure_instances.forEach((instance: KeyValuePair) => {
+      const cpus = parseInt(instance['Instance vCPUs'], 10);
+      let architecture = instance['Microarchitecture'];
+      if (!(architecture in this.azureList)) {
+        architecture = 'Average';
+      }
+      this.computeInstances['azure'][instance['Virtual Machine']] = {
+        consumption: {
+          minWatts: this.azureList[architecture]['Min Watts'] * cpus,
+          maxWatts: this.azureList[architecture]['Max Watts'] * cpus,
+        },
+        name: instance['Virtual Machine'],
+        vCPUs: instance['Instance vCPUs'],
+        maxvCPUs: parseInt(
+          instance['Platform vCPUs (highest vCPU possible)'],
+          10
+        ),
+      } as IComputeInstance;
+    });
+    aws_embodied.forEach((instance: KeyValuePair) => {
+      this.computeInstances['aws'][instance['type']].embodiedEmission =
+        instance['total'];
+    });
+    gcp_embodied.forEach((instance: KeyValuePair) => {
+      this.computeInstances['gcp'][instance['type']].embodiedEmission =
+        instance['total'];
+    });
+    azure_embodied.forEach((instance: KeyValuePair) => {
+      this.computeInstances['azure'][instance['type']].embodiedEmission =
+        instance['total'];
+    });
+  }
+
+  // Architecture strings are different between Instances-Use.JSON and the bundled Typescript from CCF.
+  // This function resolves the differences.
+  private resolveAwsArchitecture(architecture: string) {
+    if (architecture.includes('AMD ')) {
+      architecture = architecture.substring(4);
```

Comment:
after re-assigning I think all the other cases will not pass, so we can return to not execute other if statements

Reply:
@narekhovhannisyan the requirement is to realign the mismatched ones. 

There will be AMD Epyc and then we just need to cut off "AMD " from the string. 

Intel / Graviton will not come after that.